src/redux/getApiCaller.js

```diff
@@ -11,20 +11,20 @@ export default async (url, method = 'get', authorization) => {
   if (authorization) {
     headers = {
       'Content-Type': 'application/json',
-      auth_token: localStorage.getItem('token')
+      auth_token: localStorage.getItem('token'),
```

Comment:
let headers = { 'Content-Type': 'application/json' };
if(authorization)
 headers = {...headers, auth_token: localStorage.getItem('token')}
 
 no need of else